docker/scripts/create-admin-user.sh

```
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)